dartagnan/src/main/antlr4/LitmusC.g4

```diff
@@ -13,7 +13,7 @@ main
     ;
 
 variableDeclaratorList
-    :   LBrace (globalDeclarator Semi comment?)* RBrace (Semi)?
+    :   LBrace (globalDeclarator (Semi comment? globalDeclarator?)* Semi?)? RBrace Semi?
```

Comment:
I think you can do this:
```
   : LBrace (globalDeclarator Semi comment?)* (globalDeclarator comment?)? RBrace Semi?
```
It reads more naturally: "There is a list of declarators with semicolons and optionally a final one without semicolon".